src/types.ts

```diff
@@ -343,13 +345,18 @@ export interface RootProps {
   announce: Announce;
 }
 
-export interface BodyWrapperProps
-  extends Omit<RootProps, 'changeSortOrder' | 'rect' | 'pageInfo' | 'setPageInfo' | 'footerContainer' | 'translator'> {
+export interface TableBodyWrapperProps
```

Comment:
I am not a fan of this, is there no other way?

Reply:
The name? or the omit? I will look into the best way to extend the prop types in a new pr

Reply:
the omit.

Reply:
yeah it is a bit tricky. I might just define them independently. But that is a later problem